src/Components/Facility/TriageForm.tsx

```diff
@@ -263,159 +258,133 @@ export const TriageForm = (props: triageFormProps) => {
-                  handleSubmit(e);
-                }}
-              >
-                Proceed
-              </ButtonV2>
+        <DialogModal
+          title="Import Assets"
+          show={openModalForExistingTriage}
+          onClose={() => setOpenModalForExistingTriage(false)}
+          className="w-[48rem]"
+          fixedWidth={false}
+        >
+          <div className="h-screen w-full absolute flex items-center justify-center bg-modal">
+            <div className="bg-white rounded shadow p-8 m-4 max-h-full text-center flex flex-col max-w-lg w-2/3 min-w-max-content">
+              <div className="mb-4">
+                <i className="fa-solid fa-triangle-exclamation text-red-500 fa-4x"></i>
+                <h1 className="sm:text-xl text-sm">
+                  A Triage already exist on this date
+                </h1>
+                <p className="text-base">
+                  If you wish to proceed then the existing triage will be over
+                  written!
+                </p>
+              </div>
+              <div></div>
+              <div className="flex flex-col-reverse md:flex-row gap-2 mt-4 justify-end">
+                <ButtonV2
+                  variant="secondary"
+                  onClick={() => {
+                    setOpenModalForExistingTriage(false);
+                  }}
+                >
+                  Cancel
+                </ButtonV2>
+                <ButtonV2
+                  variant="danger"
+                  id="triageConfirm"
+                  onClick={(e) => {
+                    handleSubmit(e);
+                  }}
+                >
+                  Proceed
+                </ButtonV2>
+              </div>
             </div>
           </div>
-        </div>
-      </Modal>
+        </DialogModal>
+      </Page>
 
       <div className="mt-4">
         <Card>
           <form onSubmit={(e) => handleSubmit(e)}>
-            <CardContent>
-              <div className="max-w-[250px] pb-4">
-                <DateFormField
-                  required
-                  name="entry_date"
-                  label="Entry Date"
-                  value={state.form.entry_date}
-                  disableFuture
+            <div className="max-w-[250px] pb-4">
+              <DateFormField
+                required
+                name="entry_date"
+                label="Entry Date"
+                value={state.form.entry_date}
+                disableFuture
+                onChange={handleFormFieldChange}
+                position="LEFT"
+                placeholder="Entry Date"
+                error={state.errors.entry_date}
+              />
+            </div>
+            <div className="mt-2 grid gap-4 grid-cols-1 md:grid-cols-2">
+              <div>
+                <FieldLabel id="num-patients-visited-label">
+                  Patients Visited in Triage
+                </FieldLabel>
+                <TextFormField
+                  name="num_patients_visited"
```

Comment:
Make use of the `label` prop of `FormField` components to add labels instead of using `FieldLabel` separately.

This is to make designs consistent and easy to maintain.

```suggestion
                <TextFormField
                  name="num_patients_visited"
                  label="Patients Visited in Triage"
```